torch_geometric/explain/utils/metrics.py

```diff
@@ -0,0 +1,63 @@
+import torch
+from torch import Tensor
+
+
+def get_characterization_score(weights: Tensor,
+                               fidelities: Tensor) -> 'Tensor':
+    r"""Returns the componentwise characterization score of
+    fidelities[0] and fidelities[1]
```

Comment:
Would be nice to add here a bit more detail: do the two rows corresponding to fid+ and fid-? Which one is which?
And does the column corresponds to num nodes (explanation)?

Reply:
I have added explanation to which row corresponds to what.
As for the columns I'm unsure but shouldn't it be the fidelities of different explanations? As far as I understand it the two fidelities are computed as the sum average over all nodes and serve as a way to gauge the "explanatory power" of different explanations, or am I mistaken here?